package/CHANGELOG

```diff
@@ -37,6 +37,8 @@ Fixes
   * Fix hbond_analysis cannot deal with Universe where no two atoms are with 3A.
     (PR #1325)
   * Fix PDBParser docs for conect (issue #1246)
+  * Fixed bug where amber topology files would fail to load if number of atoms was
```

Comment:
You should add your github handle on line 18 with the other contributors to this version